src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -32,16 +34,28 @@ const getActiveLinkIdx = (pathname: string, links: SidebarLink[]) => {
   });
 };
 
-export const Sidebar = ({ basePath }: SidebarProps) => {
+export const Sidebar = ({ basePath, isSidebarOpen, changeSidebarState }: SidebarProps) => {
```

Comment:
Remove this props - this should be sidebar inner logic

Reply:
Logic has been moved inside the component